source/mongodb/data-access.txt

```diff
@@ -0,0 +1,113 @@
+.. _client-data-access:
+
+============================
+Query Atlas from Client Apps
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can query data in linked MongoDB Atlas clusters from client applications
+using standard MongoDB query language (MQL) syntax with **MongoDB Data Access**
+in the Realm SDKs.
+
+The MongoDB Data Access secures data with role-based App Services rules, schemas,
+and filters. These features let construct complex client-side queries with MQL
+while your data remains secure on the server.
+
+When to Use MongoDB Data Access
+-------------------------------
+
+Some general scenarios when you might want to use MongoDB Data Access in an application:
+
+- You want to use an API like the MongoDB drivers
+- You want to perform aggregations
+- You want to dynamically construct queries from the client device.
```

Comment:
Does "dynamically construct" here mean "I can add predicates to my query filter at runtime" or "I can write ad hoc queries in my client code"?

If the first, maybe we could say like "run complex queries based on user input" or something.

If the second, I think "ad hoc queries" is a succinct term we could use

Reply:
the 2nd was the intention. tho i guess technically the first is also true. will update to use the phrase ad hoc.